javascript/javascript1/week3/smartphone.js

```diff
@@ -0,0 +1,47 @@
+////////////////////Adding activities////////////////////
+const activities = [];
+
+function addActivity(date, activity, duration) {
+  activities.push({
+    date: date,
+    activity: activity,
+    duration: duration
+  });
+}
+addActivity("23/7-18", "Youtube", 30);
+addActivity("22/7-18", "Facebook", 40);
+addActivity("22/7-18", "Whatsapp", 50);
+console.log(activities);
+
+///////////////////////Show my status///////////////////////////////
+let totalDuration = 0;
+
+function showStatus(activities) {
```

Comment:
this is called show status. but this in fact returns the totalduration of the activities measured in minutes. 

Either you need to change the name of the function or the functionality.

Reply:
Changed now.